Python/Product/VSInterpreters/Interpreter/CondaEnvironmentFactoryProvider.cs

```diff
@@ -313,9 +313,18 @@ class CondaEnvironmentFactoryProvider : IPythonInterpreterFactoryProvider, IDisp
                 try {
                     // deserialize the json into an object and return the version
                     var result = JsonConvert.DeserializeObject<CondaListResult>(json);
-                    if (result != null && result.Any()) {
-                        version = new Version(result.First().Version);
+                    if (result == null) {
+                        return version;
                     }
+
+                    // look for a package named "python"
+                    var pythonResult = result.FirstOrDefault(x => x.Name == "python");
```

Comment:
should we lowercase `x.Name`?

Reply:
Nah, C# properties are supposed to be PascalCase by convention. The `JsonProperty` attribute handles changing the case appropriately during serialization.